core/lib/server/render/pub/renderMarkdownWithPub.ts

```diff
@@ -290,7 +290,10 @@ const renderMarkdownWithPubPlugin: Plugin<[utils.RenderWithPubContext]> = (conte
 					if ("form" in attrs) {
 						props.href = await utils.renderFormInviteLink({
 							formSlug: expect(attrs.form),
-							userId: context.recipient.user.id,
+							userId: expect(
+								context.recipient,
+								"Used a form invitation token without specifying a recipient member"
+							).user.id,
```

Comment:
What do you think about making this error a little more verbose? Something like `'You can't use :link{form="slug"} directive with an email address. You must set the "Recipient member" field.'`

If you're open to it I'd also like similarly explicit messages for the other two token errors in `renderWithPubUtils`, specifying which token and being very explicit about what form field needs to be set.

Reply:
Done!